packages/libs/error-stack/Cargo.toml

```diff
@@ -25,6 +34,8 @@ futures = ["dep:pin-project"]
 
 [package.metadata.docs.rs]
 all-features = true
+cargo-args = ["-Z", "unstable-options", "-Z", "rustdoc-scrape-examples=all"]
```

Comment:
Unsure, if we want to scrape examples only or the full source code
```suggestion
cargo-args = ["-Z", "unstable-options", "-Z", "rustdoc-scrape-examples=examples"]
```

Reply:
This can be tested by 
```shell
cargo doc -Zunstable-options -Zrustdoc-scrape-examples=all --all-features --no-deps --open
```
or
```shell
cargo doc -Zunstable-options -Zrustdoc-scrape-examples=examples --all-features --no-deps --open
```

Reply:
It will search for method used inside of the repository, e.g.
![image](https://user-images.githubusercontent.com/21277928/172433709-10287ead-a391-41f0-98f4-b98542b39817.png)

`rustdoc-scrape-examples=all` will search in all files, `rustdoc-scrape-examples=examples` will only search in the `examples/` folder.

Example of `Report::attach` with `=all`:
![image](https://user-images.githubusercontent.com/21277928/172434128-40fe3348-465f-42fc-a35a-76b588709d57.png)

Reply:
I think we probably want `rustdoc-scrape-examples=examples` as I don't think the ones scraped internally from the library are going to show good practices. Especially as we have some locations with `Report<()>` internally right?

Reply:
Had the same thought as well: 130b77a192146ba24d5b2a12380ed9c9dd7022a7